src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useMemo } from 'react';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const preparedPeople = useMemo(() => {
+    return people.map(person => {
+      const father = people.find(p => p.name === person.fatherName);
```

Comment:
please try not to use names like `p`, `el`, `func` etc.

also I would add addtional check `|| null`, so if there is no father its better to keep `null` value then `undefined`

`const father = people.find(p => p.name === person.fatherName) || null;`